packages/commonwealth/test/integration/api/cacheDecorator.spec.ts

```diff
@@ -76,4 +77,63 @@ describe('Cache Decorator', () => {
     expect(res).to.have.status(500);
     expect(res).to.not.have.header('X-Cache', 'HIT');
   });
+
+  describe.only('Cosmos Cache', () => {
+    // fetch('https://commonwealth.im/cosmosAPI/juno', {
+    //   headers: {
+    //     accept: '*/*',
+    //     'accept-language': 'en-US,en;q=0.9',
+    //     'content-type': 'text/plain;charset=UTF-8',
+    //     'sec-ch-ua':
+    //       '"Google Chrome";v="111", "Not(A:Brand";v="8", "Chromium";v="111"',
+    //     'sec-ch-ua-mobile': '?0',
+    //     'sec-ch-ua-platform': '"macOS"',
+    //     'sec-fetch-dest': 'empty',
+    //     'sec-fetch-mode': 'cors',
+    //     'sec-fetch-site': 'same-origin',
+    //     cookie:
+    //       '_ga=GA1.2.162706281.1678378255; mp_993ca6dd7df2ccdc2a5d2b116c0e18c5_mixpanel=%7B%22distinct_id%22%3A%20%22186c723c241852-0884bf8e8ada23-1f525634-16a7f0-186c723c24216a0%22%2C%22%24device_id%22%3A%20%22186c723c241852-0884bf8e8ada23-1f525634-16a7f0-186c723c24216a0%22%2C%22%24initial_referrer%22%3A%20%22%24direct%22%2C%22%24initial_referring_domain%22%3A%20%22%24direct%22%7D; _gid=GA1.2.371452887.1682449979; connect.sid=s%3A3tfQoT1wfz_sJbALFFUVNTbb0RIlUmaq.HXtX%2BtZHolz7H3pqG7kEoMCiUNHAKJuHRaidzjxZJBo; heroku-session-affinity=ADaDaANoA24IAXKeDf////8HYgAIbGliAAudFmECbAAAAAJtAAAABXdlYi4ybQAAAAV3ZWIuMWr99rt2PMTlIdjTsdvDH3ixptD7kQ__; _gat_gtag_UA_136522342_5=1; _gat_UA-136522342-5=1',
+    //     Referer: 'https://commonwealth.im/juno/proposals',
+    //     'Referrer-Policy': 'strict-origin-when-cross-origin',
+    //   },
+    //   body: '{"jsonrpc":"2.0","id":382288611593,"method":"abci_query","params":{"path":"/cosmos.gov.v1beta1.Query/Proposals","data":"0804","prove":false}}',
+    //   method: 'POST',
+    // });
+
```

Comment:
@NakulManchanda Could you help me sort out why I can't get this test to work? Every time I get 
`  status: 404,
  error: 'The server can not find the requested resource.'`

but I added juno to server-test.ts and i see it in my local.